examples/tensorflow/tensorflow-cifar-10/train.py

```diff
@@ -9,12 +9,12 @@
 #---------------------------------------------------------------------------
 # QUILT SETUP - added after import statements
 
-import quilt, quilt.vfs
+import quilt.vfs
+
 QUILT_PKG = 'asah/cifar10'
 
-print("Connecting to Quilt...")
-# force=False because otherwise local data in storage is overwritten
-quilt.install(QUILT_PKG, force=False)
+print("Ensuring {} is installed".format(QUILT_PKG))
+quilt.vfs.ensure_installed(QUILT_PKG)
```

Comment:
As I look at this, I'm wondering why ensure installed isn't just a parameter of vfs.setup() ?

Reply:
Not a bad idea.  Will do.

Reply:
Oh -- well, actually, would that be a useful command api addition as well?  It seems people using quilt on the python side might want to have a quick 'install' command that doesn't overwrite their existing stuff, and that kind of convenience is nice in an API.